system/services/admin/DataManagerService.cfc

```diff
@@ -86,6 +86,17 @@ component output="false" singleton=true {
 		return ListToArray( fields );
 	}
 
+	public array function listBatchEditableFields( required string objectName ) output=false {
+		var objectAttributes = _getPresideObjectService().getObjectProperties(objectName);
+		var fields           = [];
+		for(property in objectAttributes){
+			if(StructKeyExists(objectAttributes[property],"batcheditable")){
```

Comment:
The here needs to be

* `batcheditable` attribute present on the property _and_ set to `true`
* field has no unique indexes
* field is not a `one-to-many` field

At the moment the code is just checking for the presence of the `batchEditble` attribute. Please add the extra logic.